docs/tutorials/AddRemark.md

```diff
@@ -118,7 +118,6 @@ public class RemarkCommand extends Command {
 
```

Comment:
The commit message can be short, as this change doesn't really require a justification. 

```
The style of the #equals(Object) implementation varies
across the code base. Let's standardize it to improve the
consistency of the code.

Co-authored-by: Zhaoqi <e0725180@u.nus.edu>
```

In general, avoid the following when writing commit messages:

* Using `should do X` as if it's a universally accepted rule. Often, it is just our choice, and there could be ways of doing it. Instead, explain it as what the commit does `Let's do X`.
* Reversing the solution and stating it as a problem. e.g., `the code doesn't omit X` (problem) ... `let's omit X` (solution). Often the problem is not an exact opposite of the solution. e.g., Logger is giving a warning when it should not (problem) ... Let's omit X from checkstyle (solution).